api/comment_threads.rb

```diff
@@ -1,21 +1,22 @@
 
 get "#{APIPREFIX}/threads" do # retrieve threads by course
+
   threads = CommentThread.where({"course_id" => params["course_id"]})
```

Comment:
Go for it. I did some quick Googling and found little to support any claims that single quotes are more performant than double quotes.